docker/compose/mozdef_bootstrap/files/initial_setup.py

```diff
@@ -103,8 +104,15 @@
 if kibana_index_name not in all_indices:
     print "Creating " + kibana_index_name
     client.create_index(kibana_index_name)
+
+time.sleep(1)
```

Comment:
I'd worry about this not being enough to ensure that the index you just created will be available below. You might want to instead do a try except loop over some period of time with your queries

Reply:
I added some logic to wait 3 times for the index to be ready at  https://github.com/mozilla/MozDef/pull/1042/commits/18f9b7e9e47b95ee8d718ab87f68c40a4257d4bd

Reply:
Looks good